lib/stb_image.ex

```diff
@@ -140,6 +139,24 @@ defmodule StbImage do
     StbImage.Nif.to_file(filename, extension, data, height, width, channels)
   end
 
+  @doc """
+  Encodes image to an in-memory binary.
+
+  ## Example
+
+      {:ok, buffer} = StbImage.to_memory(:png, img, height, width, channels)
+
+  """
+  def to_memory(extension, data, height, width, channels)
```

Comment:
This is for consistency with `from_memory`, but I think we should rename these. My first intuition would be `to_binary`/`from_binary`, however those functions both take and return binaries, so it's ambiguous. Another option is `encode`/`decode`.

Reply:
Well, once we have the struct, `to_binary` and `from_binary` may be less ambiguous.